### ISSUE TITLE
OpenGL debug message console spam when holding an Atlas

### ISSUE THREAD

**Issue by calloatti** [Hunternif/AntiqueAtlas#225]
MC 1.14.1
antiqueatlas-5.0.3-fabric

When holding an Atlas on hand, having the overlay shown, the console gets spammed with opengl errors until the Atlas in no longer the active item on hand. 

https://paste.ee/p/Fgm2H

**Comment by tyra314** [comment]
Do you get the same error also with only Antique Atlas (and its dependencies)?

**Comment by calloatti** [comment]
Yes.

Mods:
  antiqueatlas-5.0.3-fabric-custombuild
  fabric-api-0.3.0-pre+build.165

https://paste.ee/p/C6sXu

Sorry that was with the one I built form source where you enabled the recipe for cloning Atlases, let me try again. will post a new comment with the proper jar

**Comment by calloatti** [comment]
Mods:
antiqueatlas-5.0.3-fabric
fabric-api-0.3.0-pre+build.165

https://paste.ee/p/Ll68g

**Comment by tyra314** [comment]
I cannot reproduce this on my system, but please try whether 935826c570c6375b6718069b5363f1b5d655fd9d fixes this issue for you.

**Comment by calloatti** [comment]
Seems fixed:

https://paste.ee/p/YB50I

By the way these are my specs:

![2019-05-24_19 14 40](https://user-images.githubusercontent.com/12817862/58359142-56344a00-7e58-11e9-865e-564bd9819a4d.png)

Maybe the Intel integrated graphics was the problem and why you could not repro? Just guessing.

**Comment by tyra314** [comment]
The actual problem was a (de)obfuscation mishap for one OpenGL function that happened during the port to fabric. I've got a dedicated AMD card, so it might be a more relaxed driver or something.

I'm going to release a new build soon.
